apps/cookbook/src/app/localization/localization.component.scss

```diff
@@ -16,12 +16,9 @@ $avatar-offset: -1 * utils.size('s');
 .locale-examples {
   display: flex;
   justify-content: center;
-  gap: var(--kirby-spacing-xxxl);
-
-  @container (width < #{map.get(utils.$breakpoints, small)}) {
-    flex-direction: column;
-    align-items: center;
-  }
+  flex-wrap: wrap;
```

Comment:
Good stuff - since the container query is removed, the `container-type: inline-size;` above can be removed too!